docs/toc.js

```diff
@@ -13,7 +13,23 @@ module.exports = {
       ["Espresso (Android)", "android-espresso.md"],
       ["Selendroid (Android)", "android-selendroid.md"],
       ["Windows", "windows.md"],
-      ["Mac", "mac.md"]]],
+      ["Mac", "mac.md"],
+      ["UIAutomation (iOS)", "ios-uiautomation.md"]]],
+    ["Commands", ["commands",
+      ["Status", "status.md"],
+      ["Template", "template.md"],
```

Comment:
we probably don't want the command template in our user-facing docs, right?

Reply:
Good catch.